modules/electromagnetics/doc/content/source/auxkernels/AuxComplexHeating.md

```diff
@@ -0,0 +1,28 @@
+# AuxComplexHeating
```

Comment:
the name is good for EM, but any way we can make it more obvious?
ElectricFieldHeatingAux ? is that correct?
also Aux goes at the end by convention

Reply:
I think `ElectricFieldHeatingAux` might be to vague. I can edit it to `ConductivityHeatingAux`, since the current is represented as a function of material conductivity (i.e. $\vec{J} = \sigma \vec{E}$ ). How does that sound?

Reply:
I m not knowledgable enough
@cticenhour would be a better second opinion